inference-engine/tests/ie_test_utils/common_test_utils/ngraph_test_utils.cpp

```diff
@@ -722,23 +688,85 @@ class ReadAndCompareAttributes : public ngraph::AttributeVisitor {
         }
     }
 
+    void verify_mem_buf(
+        const std::string& name, const std::shared_ptr<ngraph::runtime::AlignedBuffer>& buffer) {
+        if (should_return()) {
+            return;
+        }
+        m_visited_attributes.insert(name);
+        const auto ref_value = m_attr_ref.get<storage::MemoryChunk>(name);
+        if (!ref_value) {
+            m_cmp_result += "missing attribute name: '" + name + "'";
+            return;
+        }
+
+        if (buffer->size() != ref_value->size() ||
+            std::memcmp(ref_value->data(), buffer->get_ptr(), ref_value->size()) != 0) {
+            m_cmp_result += "mismatch in value: '" + name + "' : look in to the mem buffer";
+            return;
+        }
+    }
+    using FunctionAccessor = ngraph::ValueAccessor<std::shared_ptr<ngraph::Function>>;
+
+    void verify_function(const std::string& name, FunctionAccessor& adapter) {
+        if (should_return()) {
+            return;
+        }
+        m_visited_attributes.insert(name);
+        const auto ref_value = m_attr_ref.get<std::shared_ptr<ngraph::Function>>(name);
+        if (!ref_value) {
+            m_cmp_result += "missing attribute name: '" + name + "'";
+            return;
+        }
+        Comparator c(m_check_flags);
+        const auto result = c.compare(*ref_value, adapter.get());
+        if (!result.valid) {
+            m_cmp_result += result.message;
+        }
+    }
+
+    void verify_else(const std::string& name, ngraph::ValueAccessor<void>& adapter) {
```

Comment:
function name is confusing, it does not provide information what it will process.

Reply:
It's not best name but for now I can't find any better.